src/Database/quiz1.ts

```diff
@@ -0,0 +1,104 @@
+import { Quiz } from "./quiz.type";
+import skillImg from "./Assets/skills.jpg"
+export const quiz1:Quiz={
+    
+        _id: "1",
+        categoryName: "Skills",
+        quizName: "Skills",
+        image:skillImg,
+        score: 0,
+        totalQuestions: 5,
+        questions: [
+          {
+            _id: "1",
```

Comment:
I think the `_Id` can be made a bit more unique coz when we render it as a list more than one element on the DOM may have the same key value. Here `skills` quiz and the first question of the quiz has the same   _id:1

Reply:
Yes u r right.